CHANGELOG.md

```diff
@@ -57,71 +58,108 @@ Later support was widened in [#2254] by [@PolyMeilex].
+- Data derive macro attribute `#[data(eq)]`, which is equivalent to `#[data(same_fn = "PartialEq::eq")]`. ([#1884] by [@Maan2003])
+- `ListIter` implementation for `OrdMap`. ([#1641] by [@Lejero])
+- `ListIter` implementations for `Arc<VecDequeue<T>>` and `(S, Arc<VecDequeue<T>>)`. ([#1639] by [@raymanfx])
+- `lens` macro support for accessing nested fields. ([#1764] by [@Maan2003])
+- `lens` and `lens_mut` methods to `LensWrap` to get references to the inner lens. ([#1744] by [@SecondFlight])
+- `Lens` implementation for `Lens` tuples of length 2-8. ([#1654] by [@Maan2003])
+- Type parameters to generated lenses of generic structs, allowing for easier lens usage. ([#1591] by [@rjwittams])
+- Derived lens items now get auto-generated documentation. ([#1696] by [@lidin])
+- `theme::SCROLLBAR_MIN_SIZE` for specifying the minimum length for any scrollbar. ([#1661] by [@Cupnfish])
+- `theme::SELECTED_TEXT_INACTIVE_BACKGROUND_COLOR` for styling text selection in widgets that aren't focused. ([#1659] by [@cmyr])
+- Windows: Dark mode detection and titlebar theming. ([#2196], [#2204] by [@dristic])
+- Windows: Use custom application icon if the resource is present. ([#2274] by [@tay64])
+- Windows: Implemented `Application::get_locale`. ([#1874] by [@dfrg])
+- macOS: Implemented save dialog file format selection. ([#1847] by [@terhechte])
+- macOS: Implemented `WindowHandle::get_scale`. ([#2297] by [@dfrg])
+- Linux: `ApplicationExt::primary_clipboard` method for getting a handle to the primary system clipboard. ([#1843], [#1873] by [@Maan2003])
+- GTK: Implemented `WindowHandle::handle_titlebar`. ([#2230] by [@Steve-xmh])
+- GTK: Support for selected menu items. ([#2251] by [@longmathemagician])
+- GTK: Multiple file selection support in the file dialogs. ([#2081] by [@Psykopear])
+- X11: `set_size`, `get_size`, `set_min_size`, `resizable`, `get_position`, `set_position`, `set_level`, and `set_window_state` implementations. ([#1785] by [@Maan2003])
+- X11: Support for clipboard. ([#1805], [#1851], [#1866] by [@psychon])
+- X11: Implemented `ApplicationExt::primary_clipboard`. ([#1867] by [@psychon])
+- X11: Now setting `WM_CLASS` property. ([#1868] by [@psychon])
+- X11: Support for DPI scaling. ([#1751] by [@Maan2003])
+- X11: Support for changing cursors. ([#1755] by [@Maan2003])
+- X11: Support for custom cursors. ([#1801] by [@psychon])
+- X11: Support for window focus events. ([#1938] by [@Maan2003]
+- X11: Support for file dialogs. ([#2153] by [@jneem])
+- X11: Implemented keyboard layout detection. ([#1779] by [@Maan2003])
+- X11: Implemented `Screen::get_monitors`. ([#1804] by [@psychon])
+- X11: Implemented `Application::get_locale`. ([#1756] by [@Maan2003])
+- Web: Implemented `Application::get_locale`. ([#1791] by [@Maan2003])
+- OpenBSD: Support for building. ([#1993] by [@klemensn])
+- FreeBSD: Support for building. ([#2249] by [@nunotexbsd])
```

Comment:
I think GitHub is a little confused from the merging into the older branch. The shown additions are already in the branch `master`. 
A local diff shows the correct lack of changes here.